### ISSUE TITLE
Cannot tab into any individual row in doc list

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#374]
Cannot tab in to any individual row in the document list when using the screen reader and so cannot be read or selected

![screen shot 2016-07-06 at 14 38 35](https://cloud.githubusercontent.com/assets/13200338/16619828/c0e1a7bc-4387-11e6-9e2a-eaf5ed53c430.png)

**Comment by hashraf88** [comment]
Now when using tab to navigate I seem to be able to select multiple row and not individual rows. Notice dashed box around multipe rows.

<img width="1295" alt="screen shot 2016-09-07 at 16 09 40" src="https://cloud.githubusercontent.com/assets/13200338/18317354/c1c42334-7515-11e6-99ca-8fa9fbfc155b.png">

**Comment by VitoAlbano** [comment]
Which browser are you using? 
On my the tab is showed in this way
<img width="1433" alt="screen shot 2016-09-07 at 16 46 28" src="https://cloud.githubusercontent.com/assets/7974125/18318715/b56b6318-751a-11e6-8dee-3cbf8c76e3a8.png">

**Comment by hashraf88** [comment]
Firefox, Chrome seems to be ok
